app/experimenter/normandy/tasks.py

```diff
@@ -163,6 +165,17 @@ def update_population_percent(experiment, recipe_data):
             experiment.save()
 
 
+def update_is_high_population(experiment, recipe_data):
+    recipe_args = recipe_data.get("arguments", {})
+    if recipe_data and ("isHighPopulation" in recipe_args):
+        recipe_is_high_population = recipe_args["isHighPopulation"]
+        if (experiment.is_high_population is None) or (
+            experiment.is_high_population == recipe_is_high_population
+        ):
+            experiment.is_high_population = recipe_args["isHighPopulation"]
+            experiment.save()
```

Comment:
Just occurred to me we're not creating a changelog here, I think it's probably less relevant here than it is for something like updating the rollout percentage, since it'll get set once at the start of the recipe and theoretically not change, so probably not worth adding here, but if it comes up later that people aren't sure when/where it comes from, could be worth thinking about later?

Reply:
i did consider a changelog here, but considering nothing in the frontend was changing i thought it might be a little confusing. But that being said, I think i added a logging statement here instead